.github/workflows/test_frontend.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: subosito/flutter-action@v2
         with:
-          flutter-version: '3.10.6'
+          flutter-version: '3.13.0'
```

Comment:
Since we're bumping, why not the latest? 3.13.7

Reply:
Now I understand what's going on. I'm using flutter snap. I thought a snap refresh would mean I get the latest flutter version. Turns out no, a refresh just refreshes the snap but not flutter itself. To upgrade flutter I'd need to run flutter upgrade. This makes sense as we wouldn't want snap automatically upgrading flutter versions.